examples/src/index.js

```diff
@@ -11,13 +11,36 @@ import {
 
 const App = () => (
   <Wizard defaultStep="foo">
-    <Step id="foo">
-      <span>Foo</span>
-    </Step>
-
-    <Step id="bar">
-      <span>Bar</span>
-    </Step>
+    {({
```

Comment:
wtf

Reply:
I'm fine with adding that, but I think we need to make it possible for implementations to access the necessary props to handle the controls themselves.

Reply:
Just as long as it's optional. IE: do a check to see if child is function before calling it